src/ooni/utils.cpp

```diff
@@ -24,12 +24,8 @@ void resolver_lookup(Callback<Error, std::string> callback, Settings settings,
 }
 
 IPLocation::IPLocation(std::string path_country, std::string path_asn) {
-    if (path_asn != "") {
-        gi_asn = GeoIP_open(path_asn.c_str(), GEOIP_MEMORY_CACHE);
-    }
-    if (path_country != "") {
-        gi_country = GeoIP_open(path_country.c_str(), GEOIP_MEMORY_CACHE);
-    }
+    _path_asn = path_asn;
+    _path_country = path_country;
```

Comment:
Could you use, instead, `path_asn_`? The leading underscore is a Pythonism, but in C++ it is more common either to put a trailing underscore or to prefix `m_` to the variable. (As an aside, I think it would be much better to do `this->variable`, but this seems to be a strange thing to do in C++, so not using it.)